lapy/utils/tests/test_tet_mesh.py

```diff
@@ -211,7 +203,7 @@ def test_correct_orientations_and_boundary(tet_mesh_fixture):
 
 def test_boundary_surface_volume(tet_mesh_fixture):
     """
-    Testing computation of volume for the boundary surface mesh
+    Test orientation correction and boundary surface orientation.
```

Comment:
it this correct? the function name indicates that volume is tested not orientation?